security/middleware.py

```diff
@@ -950,19 +966,26 @@ class StrictTransportSecurityMiddleware(MiddlewareMixin):
      <https://datatracker.ietf.org/doc/rfc6797/>`_
     - `Preloaded HSTS sites <http://www.chromium.org/sts>`_
     """
+
     def __init__(self, get_response=None):
-        warnings.warn((
-            'DEPRECATED: The middleware "{name}" will no longer be '
-            'supported in future releases of this library. Refer to {url} for '
-            'an alternative approach with regards to the settings: {settings}'
-        ).format(
-            name=self.__class__.__name__,
-            url=DJANGO_SECURITY_MIDDLEWARE_URL,
-            settings=", ".join([
-                "SECURE_HSTS_SECONDS",
-                "SECURE_HSTS_INCLUDE_SUBDOMAINS",
-                "SECURE_HSTS_PRELOAD",
-            ])))
+        warnings.warn(
```

Comment:
Let's drop this one too

Reply:
Updated - https://github.com/sdelements/django-security/pull/97/commits/a3af3b829cc03630cc8cf49cdca32c827c1a2f7c